src/utils.js

```diff
@@ -549,6 +549,70 @@ function reduceConsecutiveBackslashes(str) {
 	return result;
 }
 
+/**
+ * Helper function to suggest a corrected format for the user provided input date
+ * @param {string} inputDate
+ */
+/*function suggestCorrectedDateFormat(inputDate) {
+	// Remove any non-numeric characters except 'T' and 'Z'
+	let correctedDate = inputDate.replace(/[^\dTZ]/g, '');
+
+	// If "T" is missing, insert it between the date and time
+	if (!/T/.test(correctedDate) && correctedDate.length >= 15) {
+		correctedDate = correctedDate.slice(0, 8) + 'T' + correctedDate.slice(8);
+	}
+
+	// Add missing characters to match the correct format
+	correctedDate = correctedDate.replace(
+		/(\d{4})(\d{2})(\d{2})T?(\d{2})(\d{2})(\d{2})Z?/,
+		'$1-$2-$3T$4:$5:$6Z',
+	);
+
+	return correctedDate;
+}*/
+
+function suggestCorrectedDateFormat(inputDate) {
+	// Remove any non-numeric characters except 'T' and 'Z'
+	let correctedDate = inputDate.replace(/[^\dTZ]/g, '');
+
+	// If "T" is missing, insert it between the date and time
```

Comment:
Confirming that we're not testing for a missing `Z` as well?

Reply:
In case of a missing `Z` like in this case for startTime, 
`aio api-mesh log-get-bulk --startTime 2024-08-29T23:45:56 --endTime 2024-12-49T23:16:37Z --filename mesh_logs_test.csv`
It provides the suggestion to the user like so:
`Error: Use the format YYYY-MM-DDTHH:MM:SSZ for startTime. Did you mean 2024-08-29T23:45:56Z?`

Reply:
Oh I see. You check for a missing `T` so you can be sure that the slice indexes are correct. The `Z` is always at the end so it doesn't need an explicit check 👍